### PR TITLE
add UnknownCommand and improve message parse

### PR DESCRIPTION
**Pull request by DarkPark**

### CODE REVIEW

File: src/constants/hardwareTypes.ts [hunk 1]
```diff
@@ -1,41 +1,83 @@
...
+ *
+ * 2 channels
+ */
 export const NOVATOR = 4;
 
-// RM with 2 ports, no longer in production
+/**
+ * RM with 2 ports, no longer in production.
+ *
+ * 2 channels
```
Comment (DeadByElpy) [inline src/constants/hardwareTypes.ts:26]: important to know amount of channels of hardware. Rework this or new constants?

File: src/utils/message.ts [hunk 2]
```diff
@@ -59,42 +64,41 @@ const calculateLrc = ( data: Uint8Array, initialLrc = 0x55 ) => {
...
 
-        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-        if ( !command ) {
-            throw new Error(`Unsupported command with id: ${id}.`);
-        }
+    const downlinkCommand = downlinkCommandsById[id];
+    const uplinkCommand = uplinkCommandsById[id];
+
+    // check command availability
+    if (
```
Comment (DeadByElpy) [inline src/utils/message.ts:76]:
```suggestion
    if ( !downlinkCommand && !uplinkCommand ) { return new UnknownCommand({id, data}); }
```

else if ( direction === DIRECTION_TYPE_DOWNLINK && !downlinkCommand ) return UnknownDownlinkCommand or throw an Exception immediately

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: src/CommandBinaryBuffer.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ export interface IBatteryVoltage {
      *
      * 4095 === undefined
      */
-    low: number | undefined
+    low: number | undefined,
```
[the remark this answers is not in the document]
Comment (DarkPark) [reply src/CommandBinaryBuffer.ts:11]: ts suggest to use either `;` or `,`

File: src/constants/hardwareTypes.ts (hunk 1 above)
Comment (DarkPark) [reply src/constants/hardwareTypes.ts:26]: mayb leter, let's get more info first

File: src/utils/message.ts (hunk 2 above)
Comment (DarkPark) [reply src/utils/message.ts:76]: we don't need to separate up/down UnknownCommand

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**